Index: DisplayToCurrentTime.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/DisplayToCurrentTime.java b/DisplayToCurrentTime.java
new file mode 100644
--- /dev/null	
+++ b/DisplayToCurrentTime.java	
@@ -0,0 +1,20 @@
+public class DisplayToCurrentTime {
+    public static void main(String[] args) {
+        System.out.print("Current Time is ");
+        long totalMilliSeconds = System.currentTimeMillis();
+        /* All computers use UNIX epoch and we uses that is for current time.
+        UNIX epoch starts since 00:00 (Time) / 01.01.1970 (Date) */
+        long totalSeconds = totalMilliSeconds / 1000;
+        long currentSecond = totalSeconds % 60;
+
+        long totalMinutes = totalSeconds / 60;
+        long currentMinute = totalMinutes % 60;
+
+
+        long totalHours = totalMinutes / 60;
+        long currentHour = totalHours % 24;
+
+        System.out.println( currentHour + ":" + currentMinute + ":" + currentSecond + " GMT");
+    }
+
+}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="19" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" project-jdk-name="19" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
